app/views/manuals/index.html+new_navigation.erb

```diff
@@ -0,0 +1,15 @@
+<%=
+  render(
+    'content_for_manual',
+    presented_manual: presented_manual,
+    title: presented_manual.full_title
+  )
+%>
+<%= content_for :extra_head_content do %>
+  <%= ab_test.requested_variant.analytics_meta_tag.html_safe %>
+<% end %>
+<%= render 'govuk_component/beta_label' %>
+<%= render 'govuk_component/breadcrumbs',
+  breadcrumbs: presented_manual.taxonomy_breadcrumbs %>
+<%= render 'header', presented_manual: presented_manual %>
+<%= render 'manual', presented_manual: presented_manual %>
```

Comment:
Yeh, so the idea is when the new navigation is proven to be better than the existing one, we will be doing work to remove all this duplication. For now, we have to keep it as it is.

We are only rolling out the new navigation for Education content, which is the only Taxonomy we have developed so far. I don't think there will be changes to HMRC manuals, but I will check this with the team. Thanks for point it out.